app/src/main/java/com/github/se/eventradar/ui/theme/Theme.kt

```diff
@@ -16,22 +16,76 @@ import androidx.compose.ui.platform.LocalView
 import androidx.core.view.WindowCompat
 
```

Comment:
Important: I think some comments would be helpful to describe the main important colours, what they mean/what they can be used for. This way, it is easier/faster to know how to use these colours & can save time on figuring it out.
E.g. 
// Primary colors
primary = P_80,                 // Primary color, often used for main UI elements.
onPrimary = P_20,               // Text color for content on top of primary background.
(these were just generated with ChatGPT btw)
Other than that, it looks good to me! Now we can use colours consistently, thank you!